dao/src/main/java/uk/gov/moj/sdt/dao/BulkSubmissionDao.java

```diff
@@ -96,10 +91,6 @@ public IBulkSubmission getBulkSubmissionBySdtRef(final IBulkCustomer bulkCustome
         TypedQuery<BulkSubmission> typedQuery = getEntityManager().createQuery(criteriaQuery.select(root)
                 .where(sdtCustomerPredicate, sdtBulkRefPredicate, sdtDatePredicate));
 
-        try {
-            return typedQuery.getResultStream().findFirst().orElse(null);
-        } catch(NoResultException e) {
-            return null;
-        }
+        return typedQuery.getSingleResult();
```

Comment:
What's the reasoning behind this change? Previously, if nothing is found then null was returned, now an exception will be thrown. At least one instance of code that calls this expects a null response (BulkFeedbackRequestValidator.visit()), so it won't function as expected.

Reply:
reverted this change now